2024/btt-wg.html

```diff
@@ -187,6 +187,10 @@ <h3>
             <dd>
               <p>The WebDriver Bidirectional Protocol extends WebDriver by introducing a bidirectional communication mechanism; in place of the strict command/response format of WebDriver, that bidirectional communication mechanism permits events to stream from the user agent to the controlling software, better matching the evented nature of the browser DOM.</p>
             </dd>
+            <dt id="at-driver" class="spec"><a href="https://w3c.github.io/at-driver/">AT-Driver</a></dt>
+            <dd>
+              <p>AT-Driver is a remote control interface that enables control of user agents using bidirectional communication mechanisms. It provides a platform- and language-neutral wire protocol as a way for out-of-process programs to remotely instruct the behavior of assitive technologies, spesifically screen-readers.</p>
```

Comment:
- "instruct" doesn't quite work with an object like "behavior". I'd recommend either "to remotely control the behavior of assistive technologies" or "to instruct assistive technologies"
- typo: "assitive" -> "assistive"
- While we've intentionally limited AT-Driver's focus to screen readers, that's only a temporary decision. If we mention it at all, I think it's important to qualify it so that the project isn't taken to be exclusionary (as in "[...] currently focused on screen readers"). Then again, the charter may not be a good place to describe transitory aspects, so we might want to reference screen readers more neutrally (as in, "e.g. screen readers") or remove the reference entirely.